conanfile.py

```diff
@@ -17,6 +17,10 @@ class GTestConan(ConanFile):
     exports = "CMakeLists.txt"
     url="http://github.com/lasote/conan-gtest"
     license="https://github.com/google/googletest/blob/master/googletest/LICENSE"
+    
+    def config(self):
+        if self.settings.os == "Macos":
+            self.settings.compiler.libcxx = "libc++"
```

Comment:
This config method is not needed here. You need to fill your ~/.conan/conan.conf with the setting libcxx for your clang compiler, not force all users to use libc++. 

It works because with these config lines you are assigning libcxx="libc++" to ALL users that want to use this package (actually I don't know if it works), but this setting is something that depends on every clang compiler configuration. In your OSx machine you can use the standard library from apple, or use the gnu, that's the meaning of this setting.